core/src/main/java/org/apache/gravitino/credential/config/CredentialConfig.java

```diff
@@ -67,12 +69,13 @@ public class CredentialConfig extends Config {
                   false /* reserved */))
           .build();
 
-  public static final ConfigEntry<String> CREDENTIAL_PROVIDERS =
+  public static final ConfigEntry<List<String>> CREDENTIAL_PROVIDERS =
       new ConfigBuilder(CredentialConstants.CREDENTIAL_PROVIDERS)
           .doc("Credential providers, separated by comma.")
           .version(ConfigConstants.VERSION_0_8_0)
           .stringConf()
-          .create();
+          .toSequence()
+          .createWithDefault(Collections.emptyList());
```

Comment:
It's weird that changes to `List` here alone is enough? Do you need to change to other places that uses this variable, how can they pass the type check?

Reply:
The config entry defined here is not used to get providers, updated the code, please review again